Formula/gitsh.rb

```diff
@@ -22,6 +23,15 @@ def self.old_system_ruby?
   def install
     set_ruby_path
     set_architecture
+
+    if build.head?
+      system './autogen.sh'
+      system 'RUBY="$(which ruby)" ./configure'
+      system 'make release_build_tarball'
+      system 'tar -xzvf gitsh-0.14.tar.gz'
+      chdir 'gitsh-0.14'
```

Comment:
I think this whole block could be reduced to two commands:

```ruby
if build.head?
  system './autogen.sh'
  system 'bundle'
end
```

Making the tarball has some benefit to maintainers, in that it can catch packaging bugs earlier than release time, but for someone who wants to use the master branch day-to-day that seems excessive.

We explicitly shouldn't use `RUBY=$(which ruby)` outside of development, otherwise users of ruby version managers will find gitsh breaks when they remove old rubies etc.